Source/Controllers/DataControllers/SPTableData.m

```diff
@@ -1106,10 +1106,12 @@ - (BOOL) updateAccurateNumberOfRowsForCurrentTableForcingUpdate:(BOOL)alwaysUpda
 			return YES;
 		}
 
+        NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```

Comment:
Interestingly if we're doing this everywhere we're _not_ using a shared defaults group as part of the app group. This may be something to consider doing as part of a future release, perhaps migrating old preferences into the app group. Shared defaults would let us share settings with app extensions and additional executables. @Kaspik @jamesstout